src/neo/SmartContract/IInteroperable.cs

```diff
@@ -30,5 +31,17 @@ public interface IInteroperable
         /// <param name="referenceCounter">The <see cref="ReferenceCounter"/> used by the <see cref="StackItem"/>.</param>
         /// <returns>The converted <see cref="StackItem"/>.</returns>
         StackItem ToStackItem(ReferenceCounter referenceCounter);
+
+        public IInteroperable Clone()
+        {
+            IInteroperable result = (IInteroperable)FormatterServices.GetUninitializedObject(GetType());
```

Comment:
Maybe it's better to override a `newInstance` method? the object could have arguments in the constructor

Reply:
Trust me, this is the only way. And the clone function is responsible for assigning values to all fields.